### PR TITLE
[react-imgpro] initial definitions

### PR DESCRIPTION
**Pull request by echoulen**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-imgpro/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        normalize?: boolean;
+        invert?: boolean;
+        opaque?: boolean;
+        sepia?: boolean;
+        dither565?: boolean;
+        scale?: boolean;
+        scaleToFitImage?: {
+            width: number;
+            height: number;
+		};
```
Comment (RyanCavanaugh) [inline types/react-imgpro/index.d.ts]: This document is mis-formatted all over the place - please get the braces to line up, etc.

File: types/react-imgpro/tslint.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (RyanCavanaugh) [inline types/react-imgpro/tslint.json]: Please don't disable rules unless you really have to. Most of them are there for a good reason. You can have per-line disables if needed, but they should be justified. Most packages do not disable any rules.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/react-imgpro/index.d.ts (hunk 1 above)
Comment (echoulen) [reply types/react-imgpro/index.d.ts]: Thank you, I found the reason which should not indent by `tab`

**Review by echoulen — COMMENTED** [review]
Modeifed

**Comment by echoulen** [comment]
updated

**Comment by echoulen** [comment]
@RyanCavanaugh updated

**Pull request merged**